src/formatDate.js

```diff
@@ -8,7 +8,49 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const formattedToDateParts = {}; // day, month, year
+  const values = date.split(fromFormat[3]);
+  let formattedDate = '';
+
+  for (let i = 0; i < 3; i++) {
+    if (fromFormat[i] === 'YYYY' && toFormat[i] === 'YY') {
+      formattedToDateParts[2] = values[i].slice(-2);
```

Comment:
The assignment `formattedToDateParts[2]` assumes that the year will always be placed at the third position in the `toFormat` array. This might not always be the case. Consider using a more dynamic approach that doesn't rely on fixed positions.